components/StatusItem/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { Text, View, StyleSheet, Image } from 'react-native';
+import { Constants } from 'expo';
+import PropTypes from 'prop-types';
+
+
+class Icon extends Component {
+  render() {
+    if (this.props.status == 'uploaded') {
+    pic = {
+      uri: 'https://emojipedia-us.s3.amazonaws.com/thumbs/160/facebook/105/heavy-check-mark_2714.png'
+    };
+  } else {
+    pic = {
+      uri: 'http://bizbuzzcreative.com/wp-content/uploads/2016/05/Exclamation-Point.jpg'
+    };
+  }
+    return (
+      <View style={{flex: 1}} >
+        <Image source={pic} style= {style.icon}/>
+      </View>
+    );
+  }
+}
+
+class Information extends Component {
+  render() {
+    return (
+      <View style = {style.information}>
+        <Text style = {style.danceNum}> Dance {this.props.number} </Text>
```

Comment:
Let's not call the same thing both `danceNum` and `number` in different places, let's just call the prop `danceNum` so that it's (1) more specific, (2) the same as the style name